tests/test_calling_convention_analysis.py

```diff
@@ -51,32 +51,35 @@ def run_cgc(self, binary_name):
         # print "INPUT:", map(hex, tag_manager.input_functions())
         # print "OUTPUT:", map(hex, tag_manager.output_functions())
 
-    def test_fauxware(self):
-
+    def test_fauxware_i386(self):
         amd64 = archinfo.arch_from_id("amd64")
 
         args = {
             'i386': [
                 ('authenticate', SimCCCdecl(archinfo.arch_from_id('i386'), )),
             ],
+        }
+        for arch , lst in args.items():
```

Comment:
Since the dict only has a single item in it, let's remove this and pass the arguments directly.